recipes/kanidm-clients/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "kanidm-clients" %}
+{% set version = "v1.1.0rc16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kanidm/kanidm/archive/refs/tags/{{ version.replace("rc", "-rc.") }}.tar.gz
+  sha256: 26e275bf6cf19dba34a2cc0d382c22e3b3c887f15e9fe64a1c011098173a1de9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - pkg-config
+    - openssl
+    - libudev  # [linux]
+    - sysroot_linux-64 2.17  # [linux64 and cdt_name!='cos6']
+    - {{ cdt('libudev-devel') }}  # [linux and cdt_name=='cos6']
+  host:
+    - libudev  # [linux and cdt_name!='cos6']
```

Comment:
Why not use `libudev` always?

Reply:
I had problems linking against `libudev` (I don't remember the exact error message), so I took this solution from [librealsense-feedstock](https://github.com/conda-forge/librealsense-feedstock/blob/45ef8a2bda4df7f949bc52ab45351afb3039a7fa/recipe/meta.yaml).

libudev is linux only, so that selector needs to stay. As far as I understand the cos6 selector is because libudev is only packaged for cos7, while for cos6 the cdt must be used.

Reply:
Then we should build for cos7 only. There is little value in having this workaround for little value.